### PR TITLE
Drag drop functionality and some small changes

### PR DESCRIPTION
**Pull request by pragati-dev-hub**

### CODE REVIEW

File: src/components/home/StoryUpload.jsx [hunk 1]
```diff
@@ -234,7 +234,7 @@ const StoryUpload = ({ onSuccess = () => {} }) => {
                           const selectedOption = storyWorldOptions?.find(option => option._id === e.target.value);
                           setFieldValue('storyWorld', e.target.value);
                           setFieldValue('storyWorldLead', selectedOption ? selectedOption.lead_who : '');
-                          setFieldValue('storyLeadWho', selectedOption ? selectedOption.lead_who : '');
```
Comment (afreen-md) [inline src/components/home/StoryUpload.jsx:237]: why is this removed

File: src/components/home/StoryUpload.jsx [hunk 2]
```diff
@@ -262,6 +262,7 @@ const StoryUpload = ({ onSuccess = () => {} }) => {
                         name="storyWorldLead"
                         id="storyWorldLead"
                         className="bg-gray-50 w-full md:w-[35vw] border p-2 border-gray-300 text-gray-900 sm:text-md rounded-lg focus:ring-primary-600 focus:border-primary-600 block"
+                        disabled
```
Comment (afreen-md) [inline src/components/home/StoryUpload.jsx:265]: this should be disabled only if backend given name property or whatever that we are setting for the storyworld lead exists

File: src/components/home/StoryUpload.jsx (hunk 1 above)
Comment (pragati-dev-hub) [reply src/components/home/StoryUpload.jsx:237]: As You said for that we need blank value, so i Just updated it as per requirement not removed

**Review by afreen-md — APPROVED** [review]

File: src/components/home/StoryUpload.jsx (hunk 1 above)
Comment (afreen-md) [reply src/components/home/StoryUpload.jsx:237]: ok got it

File: src/components/home/StoryUpload.jsx (hunk 2 above)
Comment (pragati-dev-hub) [reply src/components/home/StoryUpload.jsx:265]: ok, I will update it like that
